test/contracts-network/colony-network.js

```diff
@@ -152,6 +152,54 @@ contract("Colony Network", accounts => {
     });
   });
 
+  describe("when creating new colonies at a specific version", () => {
+    beforeEach(async () => {
```

Comment:
I don't get it -- why are we manually setting up a resolver here?

Reply:
I wanted a separate resolver that would allow `createColony` to be called. I didn't just want to reuse the existing one so I could check it was set up with the old version rather than the new version (though I do now see I never actually put that assert in 😅